src/development/stack.yml

```diff
@@ -344,6 +350,162 @@ services:
+        - traefik.http.routers.zammad-railsserver_secure.entryPoints=web-secure
+        - traefik.http.routers.zammad-railsserver_secure.rule=Host(`zammad.${STACK_DOMAIN}`) && PathPrefix(`/api`)
+        - traefik.http.routers.zammad-railsserver_secure.tls.options=mintls13@file
+        - traefik.http.services.zammad-railsserver.loadbalancer.server.port=3000
+    command: ["zammad-railsserver"]
+
+  zammad-scheduler:
+    image: zammad/zammad-docker-compose:latest
+    restart: always
+    depends_on:
+      - zammad-init
+    environment:
+      - POSTGRESQL_HOST=postgres
+      - POSTGRESQL_USER_FILE=/run/secrets/POSTGRESQL_USER
+      - POSTGRESQL_PASS_FILE=/run/secrets/POSTGRESQL_PASS
+      - POSTGRESQL_DB=zammad
+      - ELASTICSEARCH_HOST=zammad-elasticsearch
+      - ELASTICSEARCH_PORT=9200
+      - ZAMMAD_RAILSSERVER_HOST=zammad-railsserver
+      - ZAMMAD_WEBSOCKET_HOST=zammad-websocket
+      - REDIS_URL=redis://zammad-redis:6379
+
+    secrets:
+      - source: postgres_role_service_zammad_username
+        target: POSTGRESQL_USER
+      - source: postgres_role_service_zammad_password
+        target: POSTGRESQL_PASS
+    command: ["zammad-scheduler"]
+
+  zammad-redis:
+    image: redis:7-alpine
+    restart: always
+
+  zammad-websocket:
+    image: zammad/zammad-docker-compose:latest
+    restart: always
+    depends_on:
+      - zammad-init
+    environment:
+      - POSTGRESQL_HOST=postgres
+      - POSTGRESQL_USER_FILE=/run/secrets/POSTGRESQL_USER
+      - POSTGRESQL_PASS_FILE=/run/secrets/POSTGRESQL_PASS
+      - POSTGRESQL_DB=zammad
+      - ELASTICSEARCH_HOST=zammad-elasticsearch
+      - ELASTICSEARCH_PORT=9200
+      - ZAMMAD_RAILSSERVER_HOST=zammad-railsserver
+      - ZAMMAD_WEBSOCKET_HOST=zammad-websocket
+      - REDIS_URL=redis://zammad-redis:6379
+    secrets:
+      - source: postgres_role_service_zammad_username
+        target: POSTGRESQL_USER
+      - source: postgres_role_service_zammad_password
+        target: POSTGRESQL_PASS
+    command: ["zammad-websocket"]
+    deploy:
+      labels:
+        - traefik.enable=true
+        - traefik.http.routers.zammad-websocket.entryPoints=web
+        - traefik.http.routers.zammad-websocket.middlewares=redirectscheme
+        - traefik.http.routers.zammad-websocket.rule=Host(`zammad.${STACK_DOMAIN}`) && PathPrefix(`/ws`)
+        - traefik.http.routers.zammad-websocket_secure.entryPoints=web-secure
+        - traefik.http.routers.zammad-websocket_secure.rule=Host(`zammad.${STACK_DOMAIN}`) && PathPrefix(`/ws`)
+        - traefik.http.routers.zammad-websocket_secure.tls.options=mintls13@file
+        - traefik.http.services.zammad-websocket.loadbalancer.server.port=6042
+
+  zammad-nginx:
+    image: zammad/zammad-docker-compose:latest
+    restart: always
+    depends_on:
+      - zammad-init
+      - zammad-railsserver
+      - zammad-websocket
+    environment:
+      - NGINX_SERVER_NAME=zammad.${STACK_DOMAIN}
+      - REDIS_URL=redis://zammad-redis:6379
```

Comment:
When I apply the same environment variables as for `zammad-websocket` the container starts nginx successfully. Alos, port `8080` needs to be used, not `80`.